android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java

```diff
@@ -65,10 +66,11 @@
     @BindView(R.id.imageView_github) ImageView github;
     @BindView(R.id.imageView_twitter) ImageView twitter;
     @BindView(R.id.imageView_web) ImageView website;
-    @BindView(R.id.speaker_name_title) TextView speakerName;
+    @BindView(R.id.speaker_details_title) TextView speakerName;
     @BindView(R.id.speaker_bio) TextView biography;
-
+    @BindView(R.id.speaker_details_header) ViewGroup header;
```

Comment:
@the-dagger I think we need to write a custom rule to avoid this because Butterknife BindView must not be private AFAIK.